ui/packages/consul-ui/app/services/routlet.js

```diff
@@ -128,20 +118,43 @@ export default class RoutletService extends Service {
     };
   }
 
+
+  // modelFor gets the model for Outlet specified by `name`, not the Route
+  modelFor(name) {
+    const outlet = outlets.get(name);
+    if (typeof outlet !== 'undefined') {
+      return outlet.model;
+    }
+  }
+
   addRoute(name, route) {
-    const keys = [...outlets.keys()];
-    const pos = keys.indexOf(name);
-    const key = pos + 1;
-    const outlet = outlets.get(keys[key]);
```

Comment:
Heres the same code for `outletFor` which we moved into its own method and reuse both here and in other places.